### ISSUE TITLE
Require User Confirmation to create an account

### ISSUE THREAD

**Issue by Sun-Mountain** [Sun-Mountain/lettuceMeetApp#69]
**Description and related issues -**

Turn on [devise confirmable](https://www.rubydoc.info/github/plataformatec/devise/master/Devise/Models/Confirmable#:~:text=Confirmable%20is%20responsible%20to%20verify,a%20new%20confirmation%20instruction%20request.) so that when a new user creates an account, they must confirm their account before being able to sign into the application.

**Does this need to be broken up into smaller issues?**
If it gets too big, the exact design and wording of email can be moved to another ticket.

**What is the goal or purpose of this ticket?**
To give the application more security and make it harder for bots to create accounts, or stop people from creating burner accounts.

**Mockups or Screenshots**

### Acceptance criteria

| Given | When | Then |
| ----- | ---- | ---- |
|    User   |   Signs up for account   |   They need to confirm their email before being able to sign into the application.    |
|    User   |   They did not confirm their email   |   Can request another confirmable email sent.   |

### This task is done when…

This could include:

- [x] a list of changes to be made
- [x] all acceptance criteria are met
- [x] the description of what a user might be able to accomplish once this is done
- [x] identify 508 considerations and/or recommendations

---
